python/ray/train/_internal/backend_executor.py

```diff
@@ -565,9 +565,6 @@ def initialize_session(
             from ray.train._internal.state.schema import RunStatusEnum
 
             core_context = ray.runtime_context.get_runtime_context()
-            controller_log_file_path = (
-                ray._private.worker.global_worker.get_err_file_path()
```

Comment:
would it also be okay to keep this behavior which provides a best effort log? or does it make things complicated for the front end?